src/main/java/com/jcabi/maven/plugin/AjcMojo.java

```diff
@@ -360,6 +405,19 @@ private boolean hasSourceroots() {
     }
 
     /**
+     * Copies classes into a separate directory.
+     * @param dir The target directory.
+     * @throws IOException If something goes wrong while copying a file.
```

Comment:
@amihaiemil same here, dots should not be added, next version of Qulice will prohibit that